content/appendices.md

```diff
@@ -755,6 +755,13 @@ wish to consider handling them as `u`, `r`, and `e` respectively.
 `roomid` was otherwise unchanged.
 {{% /boxes/note %}}
 
+{{% boxes/note %}}
+{{% changed-in v="1.11" %}}
+Referencing event IDs within a room identified by room alias (`r`) rather than room ID
```

Comment:
Consider changing this to:

```
{{< changed-in v="1.11" >}} Referencing event IDs within a room identified by room alias (`r`) rather than room ID
```

(note the <> rather than %), which I think will give this whole thing a more compact representation.